akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -397,6 +397,9 @@ private[akka] class ShardRegion(
 
   import context.dispatcher
   val retryTask = context.system.scheduler.schedule(retryInterval, retryInterval, self, Retry)
+  var registrationTask: Option[Cancellable] = None
+  val initRegistrationDelay: FiniteDuration = 0.1 seconds
```

Comment:
`100.millis`, trying to avoid  doubles in such things in general AFAIR

Reply:
Ok, and should I read it from config?

Reply:
No need for more knobs. I'm fine with 0.1, otherwise derive it from `retryInterval`
```
retryInterval / 2 / 2 / 2
```